test/AxiomV1StoragePf.t.sol

```diff
@@ -30,17 +30,17 @@ contract AxiomStoragePfTest is Test {
     function setUp() public {
         yulDeployer = new YulDeployer();
         // `mainnet_10_7.v0.1` is a Yul verifier for a SNARK constraining a chain of up to 1024 block headers
-        // and Merkle-ization of their block hashes as specified in `updateRecent`.        
-        axiomVerifierAddress = address(yulDeployer.deployContract("mainnet_10_7.v0.1"));
+        // and Merkle-ization of their block hashes as specified in `updateRecent`.
+        axiomVerifierAddress = address(yulDeployer.deployContract("v0/mainnet_10_7.v0.1"));
```

Comment:
Weird to use `v0.1` here.  Not a blocker since `AxiomV1StoragePf` won't be deployed.